### ISSUE TITLE
fine-tune dataset

### ISSUE THREAD

**Issue by Chenyu-Jin** [XLearning-SCU/2024-ICLR-READ#14]
Thanks for the good work!
In your paper, you mentioned Kinetics-50, with 29,204 training pairs and 2,466 test pairs. You fine-tuned the model on the training set to obtain cav_mae_ks50.pth. I would also like to fine-tune the model following your approach. However, in the README file, only the test set link is provided. Could you please provide the download link for the Kinetics-50 training set as well?

Thank you very much for your help!
